### PR TITLE
Add configurable log levels

### PR DESCRIPTION
**Pull request by danielcooperxyz**
Add configurable log levels, as illustrated below.

```
DCooper@][wasp-ingest-rtmp](main) ▶ go build -race  && ./wasp-ingest-rtmp                                                                  14:26:43
2022-03-11T14:27:01.488Z	DEBUG	wasp-ingest-rtmp/main.go:43	DEBUG
2022-03-11T14:27:01.489Z	INFO	wasp-ingest-rtmp/main.go:44	INFO
2022-03-11T14:27:01.489Z	WARN	wasp-ingest-rtmp/main.go:45	WARN
main.main
	/Users/DCooper/source/wasp-ingest-rtmp/main.go:45
runtime.main
	/usr/local/Cellar/go/1.17.8/libexec/src/runtime/proc.go:255
2022-03-11T14:27:01.489Z	ERROR	wasp-ingest-rtmp/main.go:46	ERROR
main.main
	/Users/DCooper/source/wasp-ingest-rtmp/main.go:46
runtime.main
	/usr/local/Cellar/go/1.17.8/libexec/src/runtime/proc.go:255
[DCooper@][wasp-ingest-rtmp](main) ▶ go build -race  && ENV=production LOG_LEVEL=warn ./wasp-ingest-rtmp                                    14:27:01
2022/03/11 14:27:06 setting level: warn
{"level":"warn","ts":1647008826.6301,"caller":"wasp-ingest-rtmp/main.go:45","msg":"WARN"}
{"level":"error","ts":1647008826.630257,"caller":"wasp-ingest-rtmp/main.go:46","msg":"ERROR","stacktrace":"main.main\n\t/Users/DCooper/source/wasp-ingest-rtmp/main.go:46\nruntime.main\n\t/usr/local/Cellar/go/1.17.8/libexec/src/runtime/proc.go:255"}
```

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -3,51 +3,65 @@ package main
...
+	// 		log.Fatalln(err)
+	// 	}
+	// }()
 
-	kafka.SendMessage(messageKey, messageValue, signals)
+	// kafka := services.NewKafkaService(producer)
 
-	// videoIngest := services.NewVideoIngestService()
+	// videoIngest := services.NewVideoIngestService(kafka)
 	// videoIngest.IngestVideo()
```
Comment (dblane-digicatapult) [inline main.go]: Is this meant to be commented?

Comment (danielcooperxyz) [reply main.go]: should be updated, but will also need the rtmp url branch first

File: services/kafka.go [hunk 2]
```diff
@@ -95,12 +95,12 @@ func (k *KafkaService) StartBackgroundSend(sendWaitGroup *sync.WaitGroup, shutdo
 				Payload:   base64.StdEncoding.EncodeToString(payload.Data),
 				Metadata:  "{}",
 			}
-			// log.Printf("Encoded data: %s\n", messageValue.Payload)
+			// zap.S().Infof("Encoded data: %s", messageValue.Payload)
```
Comment (dblane-digicatapult) [inline services/kafka.go]: Spurious comment?

File: services/video-ingest.go [hunk 3]
```diff
@@ -41,13 +41,13 @@ func (vs *VideoIngestService) IngestVideo(rtmpURL string) {
...
-				log.Println("nothing found")
+				zap.S().Info("nothing found")
 
 				return
 			case count != frameSize:
-				log.Printf("end of stream: %d, %s\n", count, err)
+				zap.S().Infof("end of stream: %d, %s", count, err)
 			case err != nil:
-				log.Printf("read error: %d, %s\n", count, err)
+				zap.S().Infof("read error: %d, %s", count, err)
```
Comment (dblane-digicatapult) [inline services/video-ingest.go]: This should be error instead of info?

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]
I'm not certain some of these should be info.  The ones that occur most often should probably be debug otherwise info will get very spammy. @darrylmorton-digicatapult Thoughts?

File: services/kafka.go (hunk 2 above)
Comment (danielcooperxyz) [reply services/kafka.go]: now that it is level managed, we can uncomment

**Review by dblane-digicatapult — APPROVED** [review]

**Review by dblane-digicatapult — APPROVED** [review]

**Review by darrylmorton-digicatapult — APPROVED** [review]

**Pull request merged**